plugins/templates-maven-plugin/pom.xml

```diff
@@ -145,7 +145,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-plugin-plugin</artifactId>
-        <version>3.6.0</version>
```

Comment:
Should we also update the maven plugin info from `3.6.0` to `3.9.0` lines **89** and **120**?

Reply:
Hm, it didn't complain about a version mismatch, but sure, makes sense to keep them the same. 👍